src/main/default/lwc/caseAssistFlow/caseAssistFlow.js

```diff
@@ -40,10 +40,15 @@ export default class CaseAssistFlow extends LightningElement {
     // Debounce function for "search-as-you-type" it will trigger when there is no key stroke for 500ms.
     this.debounceHandler = debounce(async () => {
       try {
+        // If you have the AnalyticsBeacon LWC in your community, it will have generated a visitorId for you by now.
```

Comment:
I think it'd make sense for the AnalyticsBeacon to have a method that uses the Storage class of CA.js
You can get the storage instance using `window.coveoanalytics.storage.getAvailableStorage()`

You could wrap this in a utility class (not a component) and then refer to this class and its function:
![image](https://user-images.githubusercontent.com/12366410/107239520-f0567000-69f6-11eb-8400-46b19235fde2.png)